### DISCUSSION TITLE
Project yaml action not working as expected

### DISCUSSION THREAD

**Discussion by hjforbes** [opensafely/documentation#331] [Q&A] [answered]
My previous analysis was able to define two local macros in STATA  
local outcome `1' 
local dataset `2'
and define the macros in project.yaml as
 run: stata-mp:latest analysis/07b_an_multivariable_cox_models_FULL.do non_covid_death W2
So the `1' was non_covid_death and the `2' was W2. 

Project yaml is now recognising the `1' as non_covid_death W2

Anyone know how I can get this to work again?

**Comment by StevenMaude** [comment, marked as answer]
This was an issue with the Stata Docker image that `opensafely` uses to run Stata code. This issue is now [fixed](https://github.com/opensafely-core/stata-docker/pull/10).
